test/depending_project.jl

```diff
@@ -0,0 +1,244 @@
+import Term.Trees: Tree
+
+# set environment variable PRINTTREE=on to visualize the project trees of the testsets
+function printTree()::Bool
+    return haskey(ENV, "PRINTTREE")
+end
```

Comment:
I would suggest something like the following to have a default of not printing like before, but also being able to explicitly turn it off with the environment variable set. Currently, "off" would also turn the printing on.
```suggestion
# set environment variable PRINTTREE=1 to visualize the project trees of the testsets
function printTree()::Bool
    return parse(Bool, get(ENV, "PRINTTREE", "0"))
end
```
The parsing only accepts numbers in this version, so "on" would need to be changed to "1"

Also nitpick: consider renaming to `print_tree()`